src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenReadonlyStructTests.cs

```diff
@@ -924,8 +924,179 @@ public override string ToString()
             // tuple type
             type = (TypeSymbol)comp.CreateTupleTypeSymbol(ImmutableArray.Create<ITypeSymbol>(comp.ObjectType, comp.ObjectType));
             Assert.False(type.IsReadOnly);
+        }
+
+        [Fact]
+        public void CorrectOverloadOfStackAllocSpanChosen()
```

Comment:
>public [](start = 8, length = 6)

Why are these tests in `CodeGenReadonlyStructTests`?

Reply:
Moved to CodeGenTests.

---
In reply to: [133257045](https://github.com/dotnet/roslyn/pull/21006#discussion_r133257045) [](ancestors = 133257045)